instrumentation/opentelemetry-instrumentation-fastapi/pyproject.toml

```diff
@@ -34,9 +34,14 @@ dependencies = [
 
 [project.optional-dependencies]
 instruments = [
-  "fastapi ~= 0.58",
+    "fastapi >= 0.58.0",
+    "fastapi-slim ~= 0.111.0",
 ]
 
+[instruments-all]
+fastapi = ">= 0.58.0"
+fastapi-slim = "~ 0.111.0"
```

Comment:
Is this used at all?

Reply:
you are right, without this, the issue could be solved as well.